lib/player.rb

```diff
@@ -0,0 +1,31 @@
+# rubocop: disable Style/GlobalVars
+class Player
+  attr_reader :name, :symbol, :turn, :turn_one, :turn_two, :choice, :winner
+  attr_writer :name, :symbol, :turn, :turn_one, :turn_two, :choice, :winner
+
```

Comment:
#### I think this will be a good example

Notice how you used attr_reader/writer for the name variable, but after reviewing the code, I find out that you only set it up on your class constructor (initialize) and then only read from in other class,

The same example could be applied for more than just the name variable but I would let you dive in a do it by yourself to make sure you guys got the point

#### I hope it's all clear now 😉

Reply:
Hello Daniel,
I am really confused now.
I have made changes according to the previous TSE....and have to roll everything back??

When a player makes a winning move, the game stops. Didn't you see it?

Reply:
The game informs the player and stops. Then, it asks the players if they want to continue playing
Isn't it enough?